pkg/printers/internalversion/printers.go

```diff
@@ -672,7 +672,18 @@ func loadBalancerStatusStringer(s api.LoadBalancerStatus, wide bool) string {
 			result = append(result, ingress[i].Hostname)
 		}
 	}
-	r := strings.Join(result, ",")
+
+	// Remove duplicate item
```

Comment:
can you move those code into a function ? also I'm not in favor of this deduce after sort, how about this one: https://play.golang.org/p/GG2RQ1ot6t. btw do we have some exist function for this ? cc @alexandercampbell